### ISSUE TITLE
Violations do not result in a CircleCI build failure

### ISSUE THREAD

**Issue by adsteel** [brigade/scss-lint#656]
Give a bash scrip that run scss during a circle ci build:

```bash
#!/bin/bash
bundle exec scss-lint
```

If style errors are output, the build will still pass.

This is for  scss_lint  version `0.43.2`

**Comment by adsteel** [comment]
I just realized this is because my script was printing the error count after the linting run. That was probably snuffing the error code returned.
